index.html

```diff
@@ -0,0 +1,264 @@
+          <li><a class="links_Nav" id="Ignite" href="#">V Ignite</a></li>
+        </ul>
+        <!-- <img src="images/PngItem_1074369.png" alt="" class="world_img" /> -->
+      </nav>
+     
+    </header>
+
+    <section class="home_page" id="home_page">
+      <div class="section_intro">
+        <h2 class="Intro">"Hello, make it bounce with Vitron"</h2>
+        <h1 class="big_title">THE VITRON COMMUNITY SAYS HELLO.</h1>
+        <p class="p_description" id="p_desckA">
+          Vitron Is a new brand enterprise based on software and hardware
+          devellopement, It's a company wich is extended over 10 contries in the
+          world.We gonna celebrate our lancemnet of the new product in the
+          market
+          <span>The Vitron 2.1 subwoofer</span> in may 2022
+        </p>
+        <h2 class="date_conf">2022.7.20(Mon)~22(Tue)</h2>
+        <h3 class="adresse">@The Liberty Warehouse,Red Hook,Brooklyn.</h3>
+      </div>
+      <section class="main_program">
+        <h2 class="main_prog_title">Main program</h2>
+        <div class="line_red"></div>
+         <div class="card_wrap">
+          <div class="card">
+            <div class="image_wrapper">
+              <img src="images/kindpng_1430403.png" alt="" class="icon" />
+            </div>
+            <h3 class="title_card">Lecture</h3>
+            <p class="p_card">
+              Listen to feedback of our wide customers to improve quality
+            </p>
+          </div>
+          <div class="card">
+            <div class="image_wrapper">
+              <img src="images/presentation.png" alt="" class="icon" />
+            </div>
+            <h3 class="title_card">Exibition</h3>
+            <p class="p_card">
+              Large public showing and presentation of our artistic products
+            </p>
+          </div>
+          <div class="card">
+            <div class="image_wrapper">
+              <img src="images/workshop_ico.png" alt="" class="icon" />
+            </div>
+            <h3 class="title_card">Forum</h3>
+            <p class="p_card">
+              Exchange ideas and discuss issues, especially important public issues
+            </p>
+          </div>
+          <div class="card">
+            <div class="image_wrapper">
+              <img src="images/workshop-icon-15.png" alt="" class="icon" />
+            </div>
+            <h3 class="title_card">Workshop</h3>
+            <p class="p_card">
+              Smart establishment where manufacturing or handicrafts are carried on.
+            </p>
+          </div>
+          <div class="card">
+            <div class="image_wrapper">
+              <img src="images/City-Ferris-Wheel-icon.png" alt="" class="icon" />
+            </div>
+            <h3 class="title_card">V Ignite</h3>
+            <p class="p_card">
+              Introducing our products in market with a big impacts to concurencies.
+            </p>
+          </div>
```

Comment:
Kindly increase the spacing between `h3` and `p` to match the Behance design.

| Your design | Behance design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/163587957-81991da1-f38f-4f57-b90f-ac25fd4f07d7.png) | ![image](https://user-images.githubusercontent.com/24703171/163587871-30e4dc6d-953b-4c41-ac6d-a31479237814.png) |